src/tests/Microsoft.PowerFx.Interpreter.Tests/Scenarios/CustomRecordTypeWithUndefinedFields.cs

```diff
@@ -48,10 +48,10 @@ public async Task TestOriginalRecordType(string expr, object expected, bool hasE
         {
             var engine = new RecalcEngine();
             var symbolTable = new SymbolTable();
-            var symbolValues = new SymbolValues();
+            var symbolValues = new SymbolValues(symbolTable);
```

Comment:
>new SymbolValues(symbolTable); [](http://example.com/codeflow?start=31&length=30)

This is  deliberate semantic breaking change. 
Symbols Values provided at Eval must be associated with the Symbol Table used during Check.